openedx/features/course_experience/views/course_dates.py

```diff
@@ -73,36 +67,3 @@ def css_dependencies(self):
         else:
             return self.get_css_dependencies('style-mobile')
 
-    def render_to_fragment(self, request, course_id=None, **kwargs):
```

Comment:
This feels like a big removal here? Should it just be the reset dates logic?

Reply:
so i added this in myself.  i had to do this so that i could override the context in the mobile view so the button would have the necessary information.  However, we are unable to get the button to show in the mobile app, so i removed all this unnecessary code.  when we have a more defined way of doing mobile dev with better debugging resources, this will most likely be added back in so the banner can show here